jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -215,6 +223,27 @@ public void databaseIsNotCreated() throws Exception {
 	}
 
 	@Test
+	public void jmxPortIsReportedAsTag() throws Exception {
+		InfluxDbWriter writer = getTestInfluxDbWriterWithReportJmxPortAsTag();
```

Comment:
A good section of this method is duplicated from `pointsAreWrittenToInfluxDb()`. There are probably two issues here:

1) the existing tests don't check a single thing, but multiple (which means that you are actually duplicating the tests here, which is not needed). The existing test shoudl be refactor so that the common behaviour is tested only once, and the specifics are tested in different tests.
2) some common behaviour (setting up the mocks and captors) are identical and should be extracted in their own methods.